### PR TITLE
Data visualization workshop

### PR DESCRIPTION
**Pull request by HansShi**
Revise the previous errors and make pull request again.

### CODE REVIEW

**Comment by bojum** [comment]
Can I close PR #137 ?

**Comment by HansShi** [comment]
> Can I close PR #137 ?

Yes please thank you.

File: Data_Visualization_Workshop.Rmd [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (skyetim) [inline Data_Visualization_Workshop.Rmd]: there should not be YAML header.

**Review by skyetim — CHANGES_REQUESTED** [review]
1. There should be only one `Rmd` file. 
2. the only one `rmd` file should start with a single # header. 
3. there should not be a yaml header at the top of that rmd file.

File: Data_Visualization_Workshop.Rmd (hunk 1 above)
Comment (samrat-halder) [reply Data_Visualization_Workshop.Rmd]: It's fixed now. Please check @skyetim

Comment (HansShi) [reply Data_Visualization_Workshop.Rmd]: I have removed the YAML header please check

**Pull request merged**